frame/compat/bla_copy_amd.c

```diff
@@ -0,0 +1,146 @@
+/*
+
+   BLIS
+   An object-based framework for developing high-performance BLAS-like
+   libraries.
+
+   Copyright (C) 2014, The University of Texas at Austin
```

Comment:
Copyright may need to be updated on newly added files.

Reply:
I've updated the copyrights.

As an aside, I've noticed in the past that AMD has been including "All rights reserved" on the copyright notice in some files. My understanding is that this language is unnecessary because it is implied by the copyright, and therefore is redundant [1].

[1] https://www.legalzoom.com/articles/the-term-all-rights-reserved-explained

Reply:
@fgvanzee , these are the specific guidelines provided by AMD legal team.